vignettes/programming_strategy.Rmd

```diff
@@ -510,6 +520,8 @@ Regarding character vectors converted from SAS files: SAS treats missing charact
 Those are imported into R as empty strings (`""`) although in nature they are missing values (`NA`). 
 All empty strings that originate like this need to be converted to proper R missing values `NA`.
 
+This is more of a R in general type of thing, less so admiral.
```

Comment:
@bms63 per your [comment](https://github.com/pharmaverse/admiral/discussions/2183#discussioncomment-7394295)

Reply:
I would not remove it. In R `NA` means "no information available". In CDISC missing can mean both "no information available" or "no"/"false". So the important point is that admiral functions always expect `NA` even if it is a missing value with the meaning of "no" or "false". Maybe we need to rephrase it.